.gitignore

```diff
@@ -1,4 +1,8 @@
+# Website packaging
 _site
 .sass-cache
 .jekyll-metadata
 vendor
+
+# Editor files
```

Comment:
```suggestion
# https://www.gitpod.io/docs/config-gitpod-file
```
Lets include a link to where people can get more information on this specific file type, rather than being generic/ambiguous.